ansible/files/home/mailer/webhook-mailer/config.yaml

```diff
@@ -62,6 +62,9 @@ domains:
       redmine-plugin-unc:
         to:
           - clear-code@ml.commit-email.info
+      redmine-knowledgebase-migrate:
```

Comment:
Could you keep this list in alphabetical order?
(Could you move this to between `redmine-daily-issue` and `redmine-plugin-delayed-job`?)

Reply:
Whoops, it was in alphabetical order.
I moved it.